micro-demos/services-yaml/demo.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
I'll use still this demo if I get a chance to introduce Kubernetes to beginners :)
But it's easy enough to write the service spec from scratch, so I'm fine leaving this out.